packages/amplication-client/src/VersionControl/BuildSteps.tsx

```diff
@@ -24,7 +24,7 @@ export const EMPTY_STEP: models.ActionStep = {
   message: "",
 };
 
-export const GENERATE_STEP_NAME = "GENERATE_APPLICATION";
+export const GENERATE_STEP_NAME = "ADD_TO_QUEUE";
```

Comment:
I am not sure this change is expected, but since I don't see a link to an issue I am not sure what id the goal.